src/api/tests/usersRoute.spec.ts

```diff
@@ -18,6 +18,7 @@ describe('Users API tests', () => {
   describe('When a user re-invites another user', () => {
     afterEach(() => {
       jest.restoreAllMocks();
+      jest.clearAllMocks();
```

Comment:
hm maybe take my other [comment](https://github.com/IABTechLab/uid2-self-serve-portal/pull/363/files#r1524060453) with a grain of salt, the same does not seem to apply here. I thought `restore` does everything that `reset` does and `reset` does everything that `clear` does, but indeed the tests fail without the `clear` here. Any idea why - am I missing something?

Reply:
it's in the info box in the docs:  

> mockFn.mockRestore() only works when the mock was created with jest.spyOn(). Thus you have to take care of restoration yourself when manually assigning jest.fn().

I had to add clearAllMocks() to reset the count on the function being called.  You were right on the other one, though

Reply:
ahh makes sense, nice one. thanks!